Source/CarthageKit/Errors.swift

```diff
@@ -71,6 +72,10 @@ public enum CarthageError {
 	/// them.
 	case NoSharedSchemes(ProjectLocator)
 
+	/// The private cartfile contains dependencies that are already present in
+	/// the main cartfile.
+	case DuplicateDependencies([(ProjectIdentifier, /*label*/ String)])
```

Comment:
Can we encapsulate this tuple's information in a custom `struct` type instead, to make it clearer?